content/blog/posts/o3de-motion-matching-2022-04.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Motion Matching in O3DE, a Data-Driven Animation Technique"
+date: 2022-05-12
+slug: blog-motionmatching
+author: Benjamin Jillich
+blog_img: "/images/blog/motion-matching/blog_img.png"
+full_img: "/images/blog/motion-matching/full_image.png"
+---
+
+Motion matching is a data-driven animation technique that is gaining popularity since games like 'The Last Of Us Part II', 'FIFA', and 'Half-Life: Alyx' have adotped it. The complexity of animation graphs has increased along with higher expectations, a greater variety of interactions between characters and their environments, and the need for more realistic animations. Motion matching is a way to animate a character in a controlled way by jumping and blending between pre-recorded animation sequences several times a second, synthesizing the animation for the character. It can be used for realistic locomotion without the need to manually create carefully timed transition points and synchronization between animation clips.
+
+We brought motion matching to O3DE's animation system EMotion FX as a blend tree node so that it can still be used and mixed with everything else available in the animation graph.
```

Comment:
```suggestion
We brought motion matching to O3DE's animation system [EMotion FX](https://docs.aws.amazon.com/lumberyard/latest/userguide/gems-system-gem-emotionfx-animation.html ) as a blend tree node so that it can still be used and mixed with everything else available in the animation graph.
```

Reply:
Used the O3DE documentation link here: https://www.o3de.org/docs/user-guide/visualization/animation/char-intro/
Or do you think it is better to use the LY link here?

Reply:
Added

Reply:
Either is good. Think its always good to provide context to system names, techniques, more uncommon terms etc for those are unfamiliar/new to the space.